app/models/contributor.rb

```diff
@@ -16,6 +16,7 @@ class Contributor < ApplicationRecord
   has_many :replied_to_requests, -> { reorder(created_at: :desc).distinct }, source: :request, through: :replies
   has_many :received_requests, -> { broadcasted.reorder(broadcasted_at: :desc).distinct }, source: :request, through: :received_messages
   has_many :notifications_as_mentioned, class_name: 'ActivityNotification', dependent: :destroy
+  has_many :photos, through: :replies
```

Comment:
I wanted a way to count the photos for the stats as it on the request. I would strip that completely out here if there is an issue with this.

Reply:
removed with abaf5972fcb2ef11efc3f0cd5809d69012f14400

Reply:
I would be curious the thoughts of @drjakob on having just the two metrics on this page. I fear that I steered you in the wrong direction. The `photos_count` comes from the `counter_culture` gem and is kept up-to-date by the two models :point_up: 
I believe that the counts would be correct and if there were any refactoring, it would be on the side of whoever merges the two classes, but even then, I don't think there would be any.